hdtv/fitxml.py

```diff
@@ -483,10 +515,13 @@
         childs of the root element. The spectrum element is still there, but now
         a child of each fit element and not longer used in hdtv for anything
         """
-        return self.RestoreFromXml_v1_2(root, sid, calibrate, refit, interactive=interactive)
+        return self.RestoreFromXml_v1_2(
+            root, sid, calibrate, refit, interactive=interactive
+        )
 
-    def RestoreFromXml_v1_0(self, root, sids=None,
-                            calibrate=False, refit=False, interactive=False):
+    def RestoreFromXml_v1_0(
```

Comment:
Codacy found an issue: [FitXml.RestoreFromXml_v1_0 is too complex (20) (MC0001)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)